src/main/webapp/scripts/tests/comment-class-test.js

```diff
@@ -0,0 +1,27 @@
+QUnit.test('Comment Class - getError (normal use)', function(assert) {
+  const comment = new Comment('normal comment');
+  const result = comment.getError();
+  const expected = false;
+  assert.equal(result, expected, "0, false; equal succeeds");
+});
+
+QUnit.test('Comment Class - getError (no comment input)', function(assert) {
+  const comment = new Comment('');
+  const result = comment.getError();
+  const expected = 'No comment inputted';
+  assert.equal(result, expected, "error message for no comment");
+});
+
+QUnit.test('Comment Class - getError (input size exceeded)', function(assert) {
+  const comment = new Comment('A'.repeat(3001));
+  const result = comment.getError();
+  const expected = 'Comment is too long. We are currently only able to analyze comments of up to approx. 3000 characters.';
+  assert.equal(result, expected, "error message for comment length exceeded");
+});
+
+QUnit.test('Comment Class - makePerspectiveRequestString (normal use)', function(assert) {
```

Comment:
makePerspectiveRequestString kinda requires that the programmer check comment.isValid first... I could give it a default comment if isValid is false but this shouldn't even be called if it's false... idk if you have a better idea, or more edge cases to test, please let me know!

Reply:
What do you mean by it requiring it? If you feed it an invalid string what happens? does it give a nice response saying what you did wrong? If so, thats not bad (I think).